src/ReservoirPriceOracle.sol

```diff
@@ -26,7 +26,7 @@ contract ReservoirPriceOracle is IPriceOracle, IReservoirPriceOracle, Owned(msg.
     using LibSort for address[];
     using FlagsLib for *;
     using QueryProcessor for ReservoirPair;
-    using Utils for *;
+    using Utils for address;
```

Comment:
following the logic of your prev suggestion, we should also not tie `sortTokens` to the address as well? It works exactly the same way as `calculateSlot`

Reply:
Yes

Reply:
fixed